node/src/components/small_network.rs

```diff
@@ -327,6 +322,15 @@ where
             incoming_limiter,
         };
 
+        if let Some(initial_validators) = maybe_initial_validators {
+            component
+                .outgoing_limiter
+                .update_validators(Default::default(), initial_validators.clone());
+            component
+                .incoming_limiter
+                .update_validators(Default::default(), initial_validators);
+        }
```

Comment:
These should probably be treated as _active_, not upcoming, i.e. the first argument of the limiter. (See comment below.)

Reply:
I believe @marc-casperlabs intended to take care of this in an upcoming PR.  After this PR, I have a follow-up to clean up this code a bit on the ContractRuntime end.